### PR TITLE
Cypress add

### PR DESCRIPTION
**Pull request by arifratul41**
Cypress for register page with github workflow included. Facing following issue even if the test case passes

image

### CODE REVIEW

File: .github/workflows/register-test.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v1
```
Comment (ishmum123) [inline .github/workflows/register-test.yml]: did you try v2?

Comment (arifratul41) [reply .github/workflows/register-test.yml]: Sorry bhai. Tried v2 and passed without an issue.

File: cypress/integration/registration-integration.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const getInputByLabel = (label) => {
+    return cy
+        .contains('label', label)
+        .invoke('attr', 'for')
+        .then((id) => {
+            cy.get('#' + id)
```
Comment (ishmum123) [inline cypress/integration/registration-integration.js:6]: This is very clever, impressed..

File: package.json [hunk 3]
```diff
@@ -12,5 +12,8 @@
     "next": "10.1.3",
     "react": "17.0.2",
     "react-dom": "17.0.2"
+  },
+  "devDependencies": {
```
Comment (ishmum123) [inline package.json:16]: not sure if adding to `devDependencies` would work in actions... let's hope

**Pull request merged**